openfisca_us/tests/baseline/demographic/person/ccdf_duration_of_care.yaml

```diff
@@ -0,0 +1,28 @@
+- name: Weekly rate
+  period: 2022
+  input:
+    ccdf_hours_per_week: 32
+    ccdf_hours_per_day: 7
+  output:
+    duration_of_care: WEEKLY
+- name: Daily rate
+  period: 2022
+  input:
+    ccdf_hours_per_week: 28
+    ccdf_hours_per_day: 7
+  output:
+    duration_of_care: DAILY
+- name: Part-day rate
+  period: 2022
+  input:
+    ccdf_hours_per_week: 25
+    ccdf_hours_per_day: 5
+  output:
+    duration_of_care: PART_DAY
+- name: Hourly rate
+  period: 2022
+  input:
+    ccdf_hours_per_week: 10
+    ccdf_hours_per_day: 2
+  output:
+    duration_of_care: HOURLY
```

Comment:
Just missing a newline at the end (made a CLI tool for automating this here: https://github.com/nikhilwoodruff/linecheck)